contribution/self-managed-k8s/setup.sh

```diff
@@ -120,3 +120,16 @@ sudo mv kustomize /usr/local/bin
 # remove downloaded files
 cd
 sudo rm -rf /tmp/build
+
+# install bpftool from sources
+
+cd $HOME
+git clone --recurse-submodules https://github.com/libbpf/bpftool.git
+cd bpftool
+git submodule update --init
+cd src
+sudo apt-get install make -y
+sudo apt install pkgconf -y
+sudo apt install build-essential -y
+sudo make install
```

Comment:
As per my undertsanding, bpftool has statically compiled releases now. It should be fine to just install latest bpftool here?

You can reference this for installing bpftool
https://github.com/kubearmor/KubeArmor/blob/3bf31234d0927bdb5d5799e6b021d5dba4690de0/Dockerfile.init#L27-L29

Reply:
I forgot this point of fetching from GitHub releases. I think the binaries will still dynamically linked but should work. Thanks for pointing this out.

Reply:
@kranurag7 https://github.com/libbpf/bpftool/releases/tag/v7.2.0 if you check the CI release notes, it does mention statically linked binaries.

I verified them manually
![image](https://github.com/kubearmor/KubeArmor/assets/47106543/3af7736e-1242-4c1b-b681-eed7e8e88e9e)

Reply:
Thanks Barun, I thought compiling this would reference LDPATH at least somewhere but as you verified that's not the case. In retrospect, I should have verified it once. :(

Reply:
No worries 😁

Reply:
Changed sources to static binaries , Working fine for NETNEXT=-1,0,1